perl/default.nix

```
@@ -50,12 +55,17 @@ perl.pkgs.toPerlModule (stdenv.mkDerivation (finalAttrs: {
     perlPackages.Test2Harness
   ];
 
-  configureFlags = [
-    "--with-dbi=${perlPackages.DBI}/${perl.libPrefix}"
-    "--with-dbd-sqlite=${perlPackages.DBDSQLite}/${perl.libPrefix}"
+  mesonFlags = [
+    "-Dversion=${(builtins.readFile ../.version)}"
+    "-Ddbi_path=${perlPackages.DBI}/${perl.libPrefix}"
+    "-Ddbd_sqlite_path=${perlPackages.DBDSQLite}/${perl.libPrefix}"
```

Check out `lib.mesonOption`
```suggestion
    (lib.mesonOption "version" (builtins.readFile ../.version))
    (lib.mesonOption "dbi_path" "${perlPackages.DBI}/${perl.libPrefix}")
    (lib.mesonOption "dbd_sqlite_path" "${perlPackages.DBDSQLite}/${perl.libPrefix}")
```